packages/core/src/components/dropdown/dropdown.tsx

```diff
@@ -357,7 +357,7 @@ export class Dropdown {
 
     const { defaultPrevented } = this.showChanged.emit(!this.show);
 
-    if (!defaultPrevented) {
+    if (!defaultPrevented && !this.show) {
```

Comment:
In this case toggle can only be prevented if the show is not already shown. This makes for a function "toggle" which change between show=true and show=false no sense.